doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
+public repositories or code within a commercial application or network function. 
+
+The strength of open-source code is the availability of code source developed by a community which maintain and improve it. 
+Open-source code integration with application source code helps to develop and produce applications faster. But, in return, 
+it can introduce security risks if a risk management DevSecOps approach is not implemented. The GSMA white paper, “Open 
+Networking & the Security of Open Source Software Deployment - Future Networks”, alerts on these risks and addresses the 
+challenges coming with open-source code usage. Amongst these risks for security, we can mention a poor quality code 
+containing security flaws, an obsolete code with known vulnerabilities, and the lack of knowledge of open source 
+communities’ branches activity. An active branch will come with bugs fixes, it will not be the case with an inactive branch. 
+The GSMA white paper develops means to mitigate these security issues.
+
+Poor code quality is a factor of risk. Open-source code advantage is its transparency, code can be inspected by tools with 
+various capabilities such as open-source software discovery and static and dynamic code analysis.
+
+Each actor in the whole chain of software production must use a dedicated internal isolated repository separated from the 
+production environment to store vetted open-source content, which can include images, but also installer and utilities. 
+These software packages must be signed and the signature verified prior to packages or images installation. 
+Access to the repository must be granted by a dedicated authorization. The code must be inspected and vulnerabilities 
+identified as described previously. After validating the software is risk free, it can be moved to the appropriate production 
+repository.
+
+SBOM
+~~~~
+
+It is crucial to identify the software components and their origins. The Software Bill of Materials (SBOM), 
+described by `US NTIA <https://www.ntia.gov/SBOM>`__ (National Telecommunications and Information Administration), 
+is an efficient tool to identify software components. The SBOM is an inventory of software components and the relationships 
+between them. NTIA describes how to establish an SBOM and provides SBOM standard data formats. In case of vulnerability 
+detected for a component, the SBOM inventory is an effective means to identify the impacted component and provide remediation.
+
+A transparent software supply chain offers benefits for vulnerabilities remediation, but also for licensing management and it 
+provides assurance of the source and integrity of components. In order to use the SBOM efficiently and spread its adoption, 
+information must be generated and shared in a standard format. This format must be machine-readable to allow automation. The 
+NTIA white paper `“Survey of existing SBOM formats and standards” <https://www.ntia.doc.gov/files/ntia/publications
```

Comment:
How about referecing the "The Minimum Elements for an SBOM" NTIA paper, which is newer and lists CycloneDX as well? I have the impression, the current favorites in the industry are SPDX, CycloneDX (in this order) and SWID is not often discussed. Please correct me, if this impression is wrong.

Reply:
@gkunz Yes, totally agree. The plan is to restructure the section and then to develop the content on SBOM. Another PR associated with issue#2793 will be created for SBOM when this one is merged.